src/index.ts

```diff
@@ -50,7 +51,7 @@ app.all("*", async (context) => {
   const headers = new Headers(filteredHeaders);
 
   if (xGatewayServiceAuthKey && xGatewayServiceAuthType === ServiceAuthType.HEADER) {
-    const value = xGatewayServiceAuthKey.toLowerCase() === "authorization" ? `Bearer ${token}` : token;
+    const value = xGatewayAuthorizationType ? `${xGatewayAuthorizationType} ${token}` : token;
     headers.append(xGatewayServiceAuthKey, value);
   } else if (xGatewayServiceAuthKey && xGatewayServiceAuthType === ServiceAuthType.QUERY) {
     url.searchParams.append(xGatewayServiceAuthKey, token);
```

Comment:
Should we do the same here?

Then it should be named

`x-gateway-service-auth-prefix`

Reply:
are you talking about `xGatewayServiceAuthKey` ?

Reply:
Have the option to prefix token if it's a query as well.